requirements/sentry.pip

```diff
@@ -1 +0,0 @@
-sentry-sdk[flask] >= 1.1.0
```

Comment:
Don't we want to move it to `install.in`? Still confused about this requirement ^^

Reply:
Yep, me too.

Propositions:
- no change (remove it) — on our part we're installing it in udata-gouvfr so...
- leave sentry.pip and document it's usage (not done, that's why I removed it)

`install.pip` does not seem right since it's an optional feature.